jinahub/encoders/text/AudioCLIPTextEncoder/audio_clip/model/clip/clip.py

```diff
@@ -197,8 +197,7 @@ def tokenize(texts: Union[str, List[str]], context_length: int = 77) -> torch.Lo
 
     for i, tokens in enumerate(all_tokens):
         if len(tokens) > context_length:
-            leks = 'albaalbaalbaalbaalbaalbaalbaalbaalbaalbaalbaalbaalbaalbaalbaalba'
-            raise RuntimeError(f"Input {leks} {len(texts[i])} {type(texts[i])} {texts[i]} is too long for context length {context_length}")
+            raise RuntimeError(f"Input {texts[i]} is too long for context length {context_length}")
```

Comment:
same here

Reply:
This was actually my change that I was using for debugging that unnoticaby made it's way to the main branch :')